src/unit-test/RecipeExpanded.test.js

```diff
@@ -0,0 +1,29 @@
+import '../setupTest.js'
+import RecipeExpanded from '../components/RecipeExpanded';
+
+
+it('renders without crashing', () => {
+  shallow(<RecipeExpanded />)
+});
+
+
+// describe('#getIngredients', () => {
```

Comment:
Needed?
Coming back? 
Delete

Reply:
Good catch. Leftover after the refactor of RecipeExpanded into separate classes.